### ISSUE TITLE
Bug/Enh Inviting Space members from the Manage Members page does not reload the page (low priority)

### ISSUE THREAD

**Issue by martin-rueegg** [humhub/humhub#6500]
### What steps will reproduce the problem?

1. Go to a Space you have admin rights
2. Got to the Manage members page: Something like
   https://humhub.local/index.php?r=space%2Fmanage%2Fmember&cguid=5396d499-20d6-4233-800b-c6c86e5fa34b
3. Make sure there are no pending invites (tabs `Members` and `Owner` are visible, but not `Pending Invites`)
4. Invite a member using the Invite button in the Space header (complete the process)
5. Repeat step 4 with an additional member

### What is the expected result?

After step 4 and 5 the `Manage Members` page should reload:
- after step 4, in order to show the new tag `Pending Invites`.
- after step 5, in order to update the counter in `Pending Invites` or the list of pending invites, in case this tab is selected.

### What do you get instead?

Just the message that the invitation has been sent. Refresh has to be done manually.

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.15
| PHP version      | 7.4
| Operating system | linux

**Comment by martin-rueegg** [comment]
Does any of the following apply here, too?
- #6507 
- #6509 
- #6513
